internal/constants/installation_preparation_status_reasons.go

```diff
@@ -0,0 +1,6 @@
+package constants
+
+const (
+	InstallationPreparationReasonNotPerformed = "Installation preparation not performed"
+	InstallationPreparationReasonSuccess      = "Installation preparation successful"
```

Comment:
please check with QE if they are verifying the reason in their automation test because they will need to update it